lib/Mojolicious/Sessions.pm

```diff
@@ -4,7 +4,7 @@ use Mojo::Base -base;
 use Mojo::JSON;
 use Mojo::Util qw(b64_decode b64_encode);
 
-has [qw(cookie_domain secure)];
+has [qw(cookie_domain secure samesite)];
```

Comment:
Attribute has no documentation or tests. Should it be activated by default too?

Reply:
> Attribute has no documentation or tests. Should it be activated by default too?
This is a tricky question. It would provide the best default security to default to Lax for the session cookie, but it also will potentially break working applications.

I will add documentation for this but it's definitely a maintainer-question on what the default should be.